packages/CollabCodeEdit-Tests.package/CCETestDummy.class/instance/helo.st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
```

Comment:
I guess that method should not be here

Reply:
this is due to our intensive testing. i think it makes sens to clean this up on master when we release the project